cfg/dist.js

```diff
@@ -26,6 +26,7 @@ let config = Object.assign({}, baseConfig, {
       'RETRY_MIN_TIMEOUT': constants.retryMinTimeoutInMiliseconds,
       'RETRY_MAX_TIMEOUT': constants.retryMaxTimeoutInMiliseconds,
       'TICKET_REFRESH_INTERVAL': constants.ticketRefreshIntervalInSeconds,
+      'AAD_INSTANCE': constants.aadInstance,
```

Comment:
Can we extract out the duplicate attributes in this and the other `cfg/*.js` files into a shared object so the structure of this object does not get out of sync?

Reply:
Looking into it...